schedule.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Add some comments to the README on scheduled scaling i.e. how the schedule file looks and a basic run through how it works.

Reply:
👍  will do.